### ISSUE TITLE
Look at Android test app in itwinjs-core.

### ISSUE THREAD

**Issue by tcobbs-bentley** [iTwin/itwinjs-core#4321]

**Comment by aruniverse** [comment]
@tcobbs-bentley not sure what the scope of this issue was. is this still needed?

**Comment by tcobbs-bentley** [comment]
Yes. (I'm actually working on it now.) The goal is to have it run display-test-app in an emulator with things configured in such a way that it automatically downloads and opens an iModel, then exits.

**Comment by tcobbs-bentley** [comment]
@aruniverse Doing this is going to require building and running the Android display-test-app as part of the CI process, similar to how the iOS dta is run as part of the CI process. As far as I know, this hasn't been done before, and will require installation of an Android SDK upack that I set up, as well as an emulator upack that I also set up. I don't know exactly how this will be done, but I am working on the assumption that it will be possible. While the iOS dta runs as part of the macOS portion of the itwinjs-core CI process, I believe that the Android dta should run as part of the Windows portion of the itwinjs-core CI process.

CC: @chuckkir
